src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/SystemModulesPlugin.java

```diff
@@ -533,7 +540,8 @@ static class SystemModulesClassGenerator {
 
         private static final int MAX_LOCAL_VARS = 256;
 
-        private final int BUILDER_VAR    = 0;
+        private final int BUILDER_VAR    = MAX_LOCAL_VARS + 1;
+        private final int DEDUP_LIST_VAR = MAX_LOCAL_VARS + 2;
```

Comment:
Same comment as BUILDER_VAR.  But this dedup variable list is passed as a parameter to the helper method.  It's already on the local slot 2.